low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -305,17 +299,16 @@ class SSPLBundleGenerate(object):
 
 if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
-        if len(sys.argv) < 3:
-            print("Unrecognized arguments: %s" % sys.argv)
-            sys.exit(1)
```

Comment:
`sys.exit(1)` should be avoided. Otherwise the call script would exit.
So I just altered the conditions.